modules/rgbd/src/fast_icp.cpp

```diff
@@ -449,7 +450,7 @@ struct GetAbInvoker : ParallelLoopBody
                 //try to optimize
                 Point3f VxN = newP.cross(oldN);
                 float ab[7] = {VxN.x, VxN.y, VxN.z, oldN.x, oldN.y, oldN.z, oldN.dot(-diff)};
-
+                localNumInliers++;
```

Comment:
Where is it used?

Reply:
Welp! nice catch. missed removing this. Had added it for some experimentation.